drivers/sx126x/sx126x_netdev.c

```diff
@@ -384,10 +385,10 @@ static int _set(netdev_t *netdev, netopt_t opt, const void *val, size_t len)
         sx126x_set_lora_crc(dev, *((const netopt_enable_t *)val) ? true : false);
         return sizeof(netopt_enable_t);
 
-    case NETOPT_RX_TIMEOUT:
-        assert(len <= sizeof(uint32_t));
-        dev->rx_timeout = *(const uint32_t *)val;
-        return sizeof(uint32_t);
+    case NETOPT_RX_SYMBOL_TIMEOUT:
+        assert(len <= sizeof(uint16_t));
```

Comment:
from netdev documentation, this should be `uint8_t`:

https://doc.riot-os.org/group__net__netopt.html#gga19e30424c1ab107c9c84dc0cb29d9906aadd8dd23500923fd80a3f5e4c452fdfa

Reply:
Done. As discussed offline, I the value was set to 2 bytes as the upper layer was setting it to be the same. This will need to wait until bugfix is done in upper layers.

Reply:
the bugfix is already there (#16604 )

Reply:
Rebased to #16604.